axum/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 # Unreleased
 
 - **change:** Update version of multer used internally for multipart ([#2433])
+- **change:** Update tokio-tungstenite to 0.21 ([#2435])
```

Comment:
You have to include a markdown reference for the PR otherwise the link won’t work.